docs/running_an_mpc_node_in_tdx_external_guide.md

```diff
@@ -240,14 +240,14 @@ Notice that some of the commands require `sudo`, so they cannot be run using the
 
 ---
 
-##### Guest OS Image
+##### Guest OS Image - Advance (optional) - build your own and/or verify measurements.    
```

Comment:
I think the new info is more confusing than the original one. Maybe we could that somewhere else, where is it actually used a bit below